geocruncher/main.py

```diff
@@ -64,6 +64,23 @@ def run_geocruncher(args):
         # TODO do something useful with output files
         print(generated_mesh_paths)
 
+    if args[1] == 'surface':

```

Comment:
Can we rename this to ``slice`` to match our other code?

Reply:
done